web/styles/compose.css

```diff
@@ -550,11 +550,6 @@
     border: 1px solid;
     display: flex;
     align-items: center;
-    /* Banners should carry at least a minimum
-       legacy font-size of 15px, and a line-height
-       of 18px. 1.2em is 18px at 15px/1em. */
-    font-size: max(15px, var(--base-font-size-px));
```

Comment:
Wiping out the font size and the line height is not the right move here. Note how wrapping banner text changes (as well as the go-to-conversation icon) in these before/after screenshots (before being `main`):

| Before | After |
| --- | --- |
| ![evy-banners-before](https://github.com/user-attachments/assets/bb19d247-0a7b-4a71-bb57-fa66e61a6148) | ![evy-banners-after](https://github.com/user-attachments/assets/1c61b1f4-4b9b-4e18-89f7-2667660364e7) |

The most likely solution, though I haven't checked for sure, is to remove the `max()` syntax and just keep the em-based units and the `--base-font-size-px` value (which might also be extended to the `font-size` declaration on `::before` below.

Reply:
Oops, thanks for catching that. It seems like it's fine to wipe out the font size here and on the `::before` though? The only difference I see in your screenshot is line-height.

I've pushed a change with adding the line-height back in.

20px
<img width="300" alt="image" src="https://github.com/user-attachments/assets/0179f94c-7f32-4ecd-88f1-352dc820af64" />

18px
<img width="300" alt="image" src="https://github.com/user-attachments/assets/89567dcc-5ce2-409b-a945-c6ed81152539" />

14px
<img width="300" alt="image" src="https://github.com/user-attachments/assets/5962d012-cc0f-42f4-ba1e-e2d0eb24ec1d" />





I also fixed the go-to-conversation icon in #32683